### ISSUE TITLE
Can barely see settings menu

### ISSUE THREAD

**Issue by Grawmpy** [fetlife/android#382]
When I click on "Settings" from the main selection menu the screen which comes up has text that is black on a dark background. You need to make the text white or lighter so the selections will be easier to read.

**Comment by Two-Finger** [comment]
What version of Android do you use?

I used to have the same issue a few releases back, but right now the text is fine (white/grey on black).
Only the checkboxes are very dark and it's hard to see if they are checked or not.

My phone is on Android 4.4.4

**Comment by Grawmpy** [comment]
I'm currently using 5.1.1 on an Alcatel Onetouch Fierce XL. I'm adding a
screen capture to let you know what I'm talking about.

**Comment by kistitan** [comment]
Appearantly this still happens on some devices.

Settings screen are generated an it seems some specific OS systems ignores (use different) styling settings.
There might be a solution just not found yet.

**Comment by kistitan** [comment]
It seemed to be a Samsung related issue that we did not find a way to workaround, however on our Samsung test devices the issue is gone, after with the latest device OS update.

Is this issue still there for others?

**Comment by kistitan** [comment]
Closing this issue for now as we cannot reproduce it anymore.
Please feel free to repoen it if it occurred again.

**Comment by Grawmpy** [comment]
You posted asking if anybody else has an issue with the settings menu still
being subdued, and I still do. I have tried both the current release and
the beta and the issue still exists on my phone. I have a TCL ALCATEL
ONETOUCH FLINT (5054O) using Android 5.1.1. I have tried both versions on
my tablet, an RCA Viking Pro 10 and the menu is not subdued but the
selection is very hard to see when it's selected. The highlighted
background is bright yellow and the text is white. While it's selected you
can't read it but when not selected the background is black and the text is
white with labels as yellow. I have attached screenshots from both.
